README.md

```diff
@@ -1,7 +1,18 @@
 # Maple V2 Audit Reports
 
+## December 2022 Release
+
 | Auditor | Report Link |
 |---|---|
 | Trail of Bits | [`2022-08-24 - Trail of Bits Report`](https://docs.google.com/viewer?url=https://github.com/maple-labs/maple-v2-audits/files/10246688/Maple.Finance.v2.-.Final.Report.-.Fixed.-.2022.pdf) |
 | Spearbit | [`2022-10-17 - Spearbit Report`](https://docs.google.com/viewer?url=https://github.com/maple-labs/maple-v2-audits/files/10223545/Maple.Finance.v2.-.Spearbit.pdf) |
 | Three Sigma | [`2022-10-24 - Three Sigma Report`](https://docs.google.com/viewer?url=https://github.com/maple-labs/maple-v2-audits/files/10223541/three-sigma_maple-finance_code-audit_v1.1.1.pdf) |
+
+<br>
+
+## June 2023 Release
+
+| Auditor | Report Link |
+|---|---|
+| Cantina | [`2023-06-05 - Cantina Report`](https://docs.google.com/viewer?url=https://github.com/maple-labs/maple-v2-audits/files/11667848/cantina-maple.pdf) |
```

Comment:
Let's go with `Spearbit Auditors via Cantina`

Would be ideal to highlight the continuity between releases and the Cantina brand is currently unknown